### PR TITLE
Release notes for 2.0.21 release

### PR DESCRIPTION
**Pull request by derekbekoe**
cc: @JasonRShaver @sptramer

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.21/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-543) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C14%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201711141947476655-543%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BSOGlrZMELrfHxcRxP3pkQ7GXNl891uVUfAlUpWUPow%3D&st=2017-11-14T19%3A45%3A34Z&se=2017-12-15T19%3A50%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.21/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-543) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C14%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201711142233346949-543%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kJQhfBpGZNl7JFQLGQ%2BklITvxQVPZxOzjP4q%2F46IED0%3D&st=2017-11-14T22%3A31%3A17Z&se=2017-12-15T22%3A36%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -16,6 +16,112 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+* Changed all wording of "agent" to "node" in AKS
+* Deprecated `--orchestrator-release` option for `acs create`
+* Changed default VM size for AKS to `Standard_D1_v2`
+* Fixed `az aks browse` on Windows
+* Fixed `az aks get-credentials` on Windows
+
+### Appservice
+
+* Added deployment source `config-zip` for webapps and function apps
+* Added `--docker-container-logging` option [1]
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: @derekbekoe Please provide context about which command(s) this option was added to.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -16,6 +16,112 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+* Removed the `--storage` option from `--web-server-logging` [2]
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: @derekbekoe Was submitted as two arguments, and could not find this `storage` option in CLI 2.0.20. More information needed.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 3]
```diff
@@ -16,6 +16,112 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+* Improved error messages for `deployment user set`
+* Added support for creating Linux function apps
+* Fixed `list-locations`
+
+### Batch
+
+* Fixed bug in pool create command when a resource ID was used with the `--image` flag
+
+### Batchai [3]
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: The listed "options" in this section need to be named.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md (hunk 1 above)
Comment (derekbekoe) [reply latest/docs-ref-conceptual/release-notes-azure-cli.md]: addressed.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md (hunk 2 above)
Comment (derekbekoe) [reply latest/docs-ref-conceptual/release-notes-azure-cli.md]: addressed.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md (hunk 3 above)
Comment (derekbekoe) [reply latest/docs-ref-conceptual/release-notes-azure-cli.md]: addressed.

**Comment by derekbekoe** [comment]
@sptramer Comments addressed.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.21/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-543) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C15%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201711150046004468-543%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Tw9w7nBMwo%2BEWvanMCug4qklURou1EhV2bnfHyM9pJA%3D&st=2017-11-15T00%3A43%3A41Z&se=2017-12-16T00%3A48%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**